app/views/students/_form.html.erb

```diff
@@ -0,0 +1,45 @@
+<%= form_with model: student do |form| %>
+  <div class="form-group">
+    <%= form.label :first_name %>
+    <%= form.text_field :first_name, class:"form-control", placeholder:"Enter firstname" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :last_name %>
+    <%= form.text_field :last_name, class:"form-control", placeholder:"Enter lastname" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :dob, "Date of Birth" %>
+    <%= form.date_field :dob, class:"form-control" %>
+  </div>
+
+  <div class="form-group">
+    <%= form.label :department %>
+    <%= form.select :department, Student::DEPTS, {}, {class: "form-control"}%>
```

Comment:
Student::DEPTS What this means here?

Reply:
This will get the DEPTS enum initialized in Student model.